src/parser/types.ts

```diff
@@ -341,6 +341,38 @@ export interface ImageComponent extends BaseIndividualComponent {
 
 /**
  * The WebsiteComponent interface is used to define the properties of a website component. A WebsiteComponent is used to render an iframe with a website inside of it. This can be used to display an external website or an html file that is located in the public folder.
+
+``` JSON
+  {
+    "type": "website",
+    "path": "path/to/website.html",
+  }
+```
+
+ * To pass a data from the config to the website, you can use the `parameters` field as below:
+
+``` JSON
+  {
+    "type": "website",
+    "path": "path/to/website.html",
+    "parameters": {
+      "barData": [0.32, 0.01, 1.2, 1.3, 0.82, 0.4, 0.3]
+    }
+  }
+```
+ * In the `website.html` file, by including `revisit-communicate.js`, you can use the `Revisit.onDataReceive` method to retrieve the data, and `Revisit.postAnswers` to send the user's responses back to the reVISit as shown in the example below:
+
+``` HTML
+  <script src="../../revisitUtilities/revisit-communicate.js"></script>
+  <script>
+    Revisit.onDataReceive((data) => {
+      const barData = data['barData'];
+      ...
+    });
+
+    Revisit.postAnswers({ answer: [userAnswer] });
+  </script>
+```
```

Comment:
This is great! Can you go a little more in depth on how "postAnswers" works? I think it's clear how we can pull data in using the onDataReceive, but what is the object that we pass to postAnswers and why? Does it always have to be  "({answer: [userAnswer]})"? I think it would be good use the postAnswers as an example within an "onClick" action to describe more explicitly how we can use it.

Reply:
I need to check on this with Jack. I believe originally I had just a single answer, but was changed later for provenance.

Reply:
Okay, just keep this updated with what you find.

Reply:
I have updated the data structure and the example to just send the answers. Also, I have a separated the method to send provenance data. I have create a separate issue for it https://github.com/revisit-studies/study/issues/321